xonsh/tools.py

```diff
@@ -2018,11 +2018,14 @@ def format_color(string, **kwargs):
         return ansi_partial_color_format(string, style=style)
 
 
-def print_color(string, **kwargs):
+def print_color(string, color=None, **kwargs):
     """Prints a string that may contain colors. This dispatched to the shell
     method of the same name. Colors will be formatted if they have not already
     been.
     """
+    if color:
+        string = f"{{{color}}}{string}{{RESET}}"
```

Comment:
Maybe consider `color.upper()` so the input can be `color='red'`?

```suggestion
        string = f"{{{color.upper()}}}{string}{{RESET}}"
```